exogui.py

```diff
@@ -719,6 +729,7 @@ def __clickProceed__(self):
         self.logger.log('\n<--------- Starting ' + self.setKey + ' Process --------->')
         collectionDir = self.guiVars['collectionDir'].get()
         conversionType = self.guiVars['conversionType'].get()
+        useLongFolderNames = True if self.guiVars['longGameFolder'].get() == 1 else False
```

Comment:
here we should force it to False if we are not in the right type of conversion.
something like
`useLongFolderNames = True if self.guiVars['longGameFolder'].get() == 1 and (conversionType == 'batocera' || conversionType == 'retrobat' # else False`